cli/lib/commands/login.js

```diff
@@ -27,6 +27,7 @@ async function login(opts) {
   current.registries = current.registries || {};
   current.registries[opts.registry] = current.registries[opts.registry] || {};
   current.registries[opts.registry].token = token;
+  current.registries[opts.registry].username = username;
```

Comment:
This currently does nothing since the payload returned by `profile.loginWeb` only contains a `token` key. This should be extended in `services/registry/handlers/auth.js`.